WTWSMS/history/characters/greek.txt

```diff
@@ -3825,4 +3862,17 @@
 	598.1.1={
 		death=yes
 	}
+}
+
+13100021={  #Danielus 4 PLRE VOL II
+   name= Danielus
```

Comment:
I'm not sure if it breaks something, but the name should be in quotation marks.

Reply:
Ok, changing it